alarm_central_station_receiver/alarm_ctl.py

```diff
@@ -38,23 +39,46 @@ def main():
 on the keypad, or with the regular arm command.
 """
 
-    parser.add_argument('command', choices=['arm', 'disarm', 'auto-arm', 'auto-disarm'],
+    parser.add_argument('command', choices=['arm', 'disarm', 'auto-arm', 'auto-disarm', 'status', 'history'],
                         help=help_text)
+    parser.add_argument('--start-index', type=int)
+    parser.add_argument('--end-index', type=int)
 
     args = parser.parse_args()
     check_running_root()
 
-    rsp, serr = send_client_msg({'command': args.command})
+    request_msg = {'command': args.command}
+
+    if args.command == 'history':
+        if args.start_index == None or args.end_index == None:
```

Comment:
Change commant to use 'offset', and 'limit' instead of start index and end index.